src/py42/clients/securitydata.py

```diff
@@ -251,20 +251,10 @@ def _stream_file(self, checksum, version_info):
             device_guid, md5_hash, sha256_hash, path
         )
         if version:
-            pds = self._storage_service_factory.create_preservation_data_service(
-                version[u"storageNodeURL"]
-            )
-            token = pds.get_download_token(
-                version[u"archiveGuid"],
-                version[u"fileId"],
-                version[u"versionTimestamp"],
-            )
-            return pds.get_file(str(token))
+            return self._get_file_stream(version)
 
         raise Py42Error(
-            u"No file with hash {} available for download on any storage node.".format(
-                checksum
-            )
+            u"No file with hash {} available for download.".format(checksum)
```

Comment:
`u` not necessary anymore

Reply:
I will inevitably have to deal with that merge conflict, so I'll handle all those changes at that time.